addons/website_sale_slides/models/sale_order.py

```diff
@@ -17,15 +17,17 @@ def _action_confirm(self):
         )
         products = so_lines.mapped('product_id')
         related_channels = self.env['slide.channel'].search(
-            [('product_id', 'in', products.ids)]
+            [('product_id', 'in', products.ids), ('enroll', '=', 'payment')],
         )
         channel_products = related_channels.mapped('product_id')
 
         channels_per_so = {sale_order: self.env['slide.channel'] for sale_order in self}
         for so_line in so_lines:
             if so_line.product_id in channel_products:
+                so_line.name = 'Access to:'
```

Comment:
Missing translation marks (+ would be better to create it using a string formatted to ease translations)

Reply:
Removed (see above)